esp/esp/users/migrations/0009_auto_20170201_0057.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0008_teacherinfo_affiliation'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
likewise not sure why this is here.  Maybe your migrations aren't up to date?  Or maybe this is a different problem?  In any case to be honest I'd just manually fix up the old migration to not have `null=True` and remove the new ones, although maybe that will do weird things to your local DB.